drivers/kscan/kscan_handlers.c

```diff
@@ -11,21 +11,21 @@ static inline int z_vrfy_kscan_config(struct device *dev,
 				      kscan_callback_t callback_isr)
 {
 	Z_OOPS(Z_SYSCALL_DRIVER_KSCAN(dev, config));
-	Z_OOPS(Z_SYSCALL_VERIFY_MSG(callback == 0,
+	Z_OOPS(Z_SYSCALL_VERIFY_MSG(callback_isr == 0,
 				    "callback cannot be set from user mode"));
-	return z_impl_kscan_config((struct device *)dev, callback);
+	return z_impl_kscan_config((struct device *)dev, callback_isr);
 }
 #include <syscalls/kscan_config_mrsh.c>
 
-static inline int z_vrfy_kscan_disable_callback(struct device *dev);
+static inline int z_vrfy_kscan_disable_callback(struct device *dev)
 {
 	Z_OOPS(Z_SYSCALL_DRIVER_KSCAN(dev, disable_callback));
 
 	return z_impl_kscan_disable_callback((struct device *)dev);
 }
 #include <syscalls/kscan_disable_callback_mrsh.c>
 
-static int z_vrfy_kscan_enable_callback(struct device *dev);
+static int z_vrfy_kscan_enable_callback(struct device *dev)
```

Comment:
The changes in this file don't appear to be related to the commit message, `tests: drivers: kscan fixes failing test due to wrong devicetree define`

Reply:
Fixed!